docs/source/java/jdbc.rst

```diff
@@ -84,8 +84,8 @@ inconsistent scale. A RoundingMode can be set to handle these cases:
      }
    }
 
-Currently, it is not possible to define a custom type conversion for a
-supported or unsupported type.
+Currently, it is not possible to override the type conversion for a
+supported type, or define a new conversion for an unsupported type.
```

Comment:
Sorry, I think we do support overriding the conversion now. Please see https://github.com/apache/arrow/blob/master/java/adapter/jdbc/src/main/java/org/apache/arrow/adapter/jdbc/JdbcToArrowConfig.java#L76

Reply:
Thanks.